madsim-rdkafka/src/std/admin.rs

```diff
@@ -334,19 +383,14 @@ impl<C: ClientContext> AdminClient<C> {
     }
 }
 
-#[async_trait::async_trait]
 impl FromClientConfig for AdminClient<DefaultClientContext> {
-    async fn from_config(config: &ClientConfig) -> KafkaResult<AdminClient<DefaultClientContext>> {
-        AdminClient::from_config_and_context(config, DefaultClientContext).await
+    fn from_config(config: &ClientConfig) -> KafkaResult<AdminClient<DefaultClientContext>> {
+        AdminClient::from_config_and_context(config, DefaultClientContext)
     }
 }
```

Comment:
We should keep these functions async for simulation?

https://github.com/madsim-rs/madsim/tree/main/madsim-rdkafka#api-modification